pkg/kubelet/cm/devicemanager/pod_devices.go

```diff
@@ -187,13 +187,13 @@ func (pdev *podDevices) toCheckpointData() []checkpoint.PodDevicesEntry {
 		for conName, resources := range containerDevices {
 			for resource, devices := range resources {
 				if devices.allocResp == nil {
-					klog.Errorf("Can't marshal allocResp for %v %v %v: allocation response is missing", podUID, conName, resource)
+					klog.ErrorS(nil, "Can't marshal allocResp, allocation response is missing", "podUID", podUID, "containerName", conName, "resourceName", resource)
```

Comment:
If there is not error, use klog.InfoS instead.

Reply:
No, this is correct as is, we don't want to change the log level.